projects/avodata/firstStage.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import math
+
+def factorial(n):
```

Comment:
Please use 2-space indentation using spaces, not tabs.

Reply:
done

Reply:
It's still tabs and not 2 spaces I believe.

Reply:
How can i change it for all the file? its make me Balagan...

Reply:
What IDE are you using?